Program.cs

```diff
@@ -1,2 +1,65 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System;
+
+namespace InventoryManagementSystem
+{
+    public enum Operation
+    {
+        Add = 1,
+        View,
+        Edit,
+        Delete,
+        Search,
+        Exit
+    }
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Inventory inventory = new Inventory();
+
+            Console.WriteLine("Welcome to the Inventory Management System!");
+
+            while (true)
+            {
+                Console.WriteLine("\nChoose an operation:");
+                Console.WriteLine("1. Add a product");
+                Console.WriteLine("2. View all products");
+                Console.WriteLine("3. Edit a product");
+                Console.WriteLine("4. Delete a product");
+                Console.WriteLine("5. Search for a product");
+                Console.WriteLine("6. Exit");
+
+                Console.Write("Enter your choice: ");
+                if (!Enum.TryParse(Console.ReadLine(), out Operation choice) || !Enum.IsDefined(typeof(Operation), choice))
```

Comment:
what do you mean by using isDefined? 
there is no need to check the choice type since the choice is already the output of the TryParse method so it definitely its enum of Operation